source/_patterns/typography/examples.hbs

```diff
@@ -21,9 +21,24 @@
 			<h6 data-variant="light">Headline 6</h6>
 		</db-tab>
 		<db-tab label="Body" name="tab-bar">
+			<h1>functional</h1>
```

Comment:
it's best to include the h1 element only once on a page (or at least within a context, like. e.g. section).